context/app/static/js/shared-styles/pages/IconPageTitle/style.ts

```diff
@@ -1,15 +1,8 @@
 import { styled } from '@mui/material/styles';
-
 import HeaderIcon from 'js/shared-styles/icons/HeaderIcon';
 
-const FlexContainer = styled('div')(({ theme }) => ({
-  display: 'flex',
-  alignItems: 'center',
-  marginBottom: theme.spacing(2),
```

Comment:
Don't we need the `marginBottom` for all of the other instances of `IconPageTitle`?

Reply:
I couldn't find any references to `IconPageTitle` outside of workspaces, so this might be a duplicate case: 
![image](https://github.com/user-attachments/assets/2a095000-434c-4995-b499-910f22159116)

In the other cases, the `SummaryData` component appears to be handling the display of this title and icon: https://github.com/hubmapconsortium/portal-ui/blob/dcc2a36dc9d44fa9dbffc702ab00c2a8d5150e2c/context/app/static/js/components/detailPage/summary/SummaryData/SummaryData.tsx#L40-L44